fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -967,6 +970,11 @@ public void finishTransaction(long transactionId, Set<Long> errorReplicaIds) thr
                                     if (!replica.getState().canLoad()) {
                                         continue;
                                     }
+                                    // if replica not commit yet, skip it. This may happen when it's just create by clone.
+                                    if (transactionState.tabletCommitInfosNotContainsReplica(tablet.getId(), 
```

Comment:
i think it's better to use proactive syntax, e.g. `!transactionState.tabletCommitInfosContainsReplica()`

Reply:
Done